packages/core/addon/components/navi-visualizations/apex-pie.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import layout from '../../templates/components/navi-visualizations/apex-pie';
+import { alias } from '@ember/object/computed';
+export default class NaviVisualizationsApexPie extends Component {
+  layout = layout;
+  @service() bardMetadata;
+
+  @alias('options.metadata.series.config.metric.metric') metric;
```

Comment:
```suggestion
  get metric() {
    return this.options.metadata.series.config.metric.metric;
  }
```
Let's avoid creating two-way data bindings